manifests/cron.pp

```diff
@@ -22,10 +26,13 @@
     $scout_roles_switch = " -r ${roles}"
   }
 
+  $command = "/usr/bin/env scout \${SCOUT_KEY}${scout_environment_switch}${scout_roles_switch}"
```

Comment:
Will look at pushing this fix upstream to puppet-stdlib as it seems mad that `undef` ends up as a string.